Source/IntuneAppBuilder/Services/IntuneAppPublishingService.cs

```diff
@@ -198,7 +198,7 @@ private async Task CreateBlobAsync(IntuneAppPackage package, MobileAppContentFil
             var blockCount = 0;
             var blockIds = new List<string>();
 
-            const int chunkSize = 5 * 1024 * 1024;
+            const int chunkSize = 25 * 1024 * 1024;
```

Comment:
I'm hoping you tested this, but are we sure that a chunk of this size works? The units aren't well defined in this code, but from a cursory Google it appears that depending on what type of blob you are creating that 25 MiB is unsupported.

Reply:
tested and works